packages/@aws-cdk/cloudformation-include/lib/cfn-include.ts

```diff
@@ -202,6 +210,20 @@ export class CfnInclude extends core.CfnElement {
     return ret;
   }
 
+  /**
+   * Returns the CfnRule object from the 'Rules'
+   * section of the CloudFormation template with the given name.
+   * Any modifications performed on that object will be reflected in the resulting CDK template.
+   *
+   * If a Rule with the given name is not present in the template,
+   * an exception will be thrown.
+   *
+   * @param ruleName the name of the Rule in the CloudFormation template
+   */
+  public getRule(ruleName: string): core.CfnRule {
+    return this.rules[ruleName];
```

Comment:
To be consistent with all of the other `get` methods, I think this needs to be:

```suggestion
    const ret = this.rules[ruleName];
    if (!ret) {
      throw new Error(`Rule with name '${ruleName}' was not found in the template`);
    }
```

Reply:
Absolutely correct. Added, plus a unit test.